lib/metasploit/framework/login_scanner/snmp.rb

```diff
@@ -17,6 +17,35 @@ class SNMP
         PRIVATE_TYPES        = [ :password ]
         REALM_KEY            = nil
 
+        # @!attribute retries
+        #   @return [Fixnum] The number of retries
+        attr_accessor :retries
+
+        validates :retries,
+                  presence: true,
+                  numericality: {
+                    only_integer:             true,
+                    greater_than_or_equal_to: 0
+                  }
+
+        # @!attribute version
```

Comment:
YARD has advised that `@!attribute` is not meant to be used on `attr_*`.  Use comments directly instead:

```ruby
# The SNMAP version to scan
#
# @return [String]
attr_accessor :version
```